.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  root: true,
+  extends: ['plugin:goodeggs/recommended', 'plugin:goodeggs/flowtype'],
+  env: {
+    node: true,
+  },
+  overrides: [
+    {
+      files: ['src/use_sinon_sandbox.js'],
+      env: {
+        mocha: true,
+      },
+    },
+    {
+      files: ['babel.config.js'],
+      rules: {
+        'import/no-commonjs': 'off',
+      },
+    },
```

Comment:
Would you mind using this exact override, for consistency and future-proofing? https://github.com/goodeggs/goodeggs-foodhubs/blob/521d3d42f3b5dcc10c65a18958e0c3cf7c14a671/.eslintrc.js#L13-L22